test/Kozani/KozaniManagerRuntimeTests/KozaniManagerRuntimeTests.cpp

```diff
@@ -14,14 +14,48 @@ namespace KozaniManagerTestPackage
     constexpr PCWSTR c_PackageFullName = L"KozaniManager_1.0.0.0_neutral__8wekyb3d8bbwe";
 }
 
+HRESULT RegisterDVCPluginIfNeeded()
+{
+    PCWSTR subkey = L"Software\\Microsoft\\Terminal Server Client\\Default\\AddIns\\KozaniDVC";
```

Comment:
NIT: `PCWSTR subkey{ L"..." };`

[ES.23: Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

Same comment applies throughout